docs/source/quickstart.md

```diff
@@ -0,0 +1,133 @@
+# Quickstart Guide
+
+This page provides a quick introduction to get you started with StreamSim.
+
+First, make sure you have StreamSim installed (see [Installation](installation.md) for details).
+
+
+## Generating Mock Stream Data
+
+StreamSim can generate parametric stellar stream models using configuration files or directly in Python.
+
+### Example 1: Using Command-Line Tools
+
+The simplest way to generate a stream is using the provided command-line scripts:
+
+```bash
+# Generate a simple linear stream
+./bin/generate_stream.py config/toy1_config.yaml -o toy1_stream.csv --plot
+
+# Generate a sinusoidal stream
+./bin/generate_stream.py config/toy2_config.yaml -o toy2_stream.csv --plot
+
+# Generate Pal 5 stream from interpolation
+./bin/generate_stream.py config/pal5_config.yaml -o pal5_stream.csv --plot
+```
+
+These commands will:
+- Read the stream parameters from the configuration file
+- Generate mock stellar positions (phi1, phi2) and magnitudes
+- Save the results to a CSV file
+- Create visualization plots (with `--plot` flag)
+
+### Example 2: Generating Streams in Python
+
+You can also generate streams programmatically:
+
+```python
+import numpy as np
+import pandas as pd
+from stream_sim.model import StreamModel
+from stream_sim.utils import parse_config
+
+# Load configuration
+config = parse_config('config/toy1_config.yaml')
+
+# Create stream model and generate stars
+stream = StreamModel(config['stream'])
+stream_df = stream.sample(config['stream']['nstars'])
+
+# The dataframe contains: phi1, phi2, distance, magnitudes, etc.
+print(stream_df.head())
+```
+
+## Converting to Observable Quantities
+
+Once you have mock stream data (either generated or from simulations), you can convert it to realistic observations using the `StreamInjector` class.
+
+### Example 3: Applying Survey Effects
+
+```python
+import numpy as np
+import pandas as pd
+from stream_sim import surveys, observed
+
+# Load a survey (e.g., LSST Year 1)
+lsst_survey = surveys.Survey.load(survey='lsst', release='yr1')
+
+# Create the stream injector
+injector = observed.StreamInjector(lsst_survey)
+
+# Create or load your mock stream data
+# Here we create a simple test dataset
+# This could contain (ra, dec) coordinates if you want to skip coordinate transformation
+rng = np.random.default_rng(42)
+mock_data = pd.DataFrame({
+    'phi1': rng.uniform(-5, 5, 1000),      # Stream longitude
+    'phi2': rng.uniform(-1, 1, 1000),      # Stream latitude
+    'mag_g': rng.uniform(18, 28, 1000),    # g-band magnitude
+    'mag_r': rng.uniform(18, 28, 1000),    # r-band magnitude
+})
+
+# Apply survey effects: footprint, extinction, photometric errors
+observed_data = injector.inject(
+    mock_data, 
+    seed=42,
+    bands = ['r', 'g'] # Choose the bands to use
+    mask_type=['footprint', 'ebv'],  # Restrict injection to a mask created from the footprind and low dust area
+    verbose=True
```

Comment:
can we specify the rotation matrix here? if so I would include R_mat = None (or whatever it is called)